safe/impact_functions/inundation/flood_polygon_population/impact_function.py

```diff
@@ -135,10 +136,11 @@ def _tabulate_action_checklist(self, table_body, total):
         table_body.append(
             TableRow(tr('Total population: %s') % format_int(total)))
         table_body.append(TableRow(self.parameters['provenance']))
-        table_body.append(
-            TableRow(
-                tr('"nodata" values in the exposure layer are treated as 0 '
-                   'when counting the affected or total population')))
+        if nan_warning:
+            table_body.append(
+                TableRow(
+                    tr('"nodata" values in the exposure layer are treated as '
```

Comment:
Adding message like others won't harm this ("nodata was carried through to impact layer..." message)